provider/azure/config_test.go

```diff
@@ -80,7 +97,9 @@ func (s *configSuite) assertConfigValid(c *gc.C, attrs testing.Attrs) {
 func (s *configSuite) assertConfigInvalid(c *gc.C, attrs testing.Attrs, expect string) {
 	cfg := makeTestModelConfig(c, attrs)
 	_, err := s.provider.Validate(cfg, nil)
-	c.Assert(err, gc.ErrorMatches, expect)
+	c.Check(err, gc.NotNil)
+	//c.Assert(err, gc.ErrorMatches, expect)
```

Comment:
For some reason, `c.Assert(err, gc.ErrorMatches, expect)` isn't working for me here. Suggestions about why are very welcome.